bitcoin-extensions/styles.css

```diff
@@ -0,0 +1,22 @@
+html {
+    background-color: lightblue;
+}
+
+
+
+#data {
+    width: 400px;
+    border: 1px dashed black;
+    font-size: 20px;
+    text-align: center;
+    margin: 0 auto;
```

Comment:
Instead of having two margin properties, you can actually combine them all into 1. 

`margin: -105px 0 0 0;`

This is in effect changing the margin for Top, right, bottom, left (in that order) on one line. You could also specify margin for top and bottom with one number and then right and left with the second number like this:

'margin: 50px 20px;'

But of course in your code you have specified different margin settings for the top and bottom, hence why you should follow the first example to condense your code.